languages/swift/exercises/concept/strings-and-characters/.docs/introduction.md

```diff
@@ -0,0 +1,111 @@
+\\0 (null character), \\\\ (backslash), \\t (horizontal tab), \\n (newline), \\r (carriage return), \\" (double quotation mark) and \\' (single quotation mark).
+
+An empty string is represented by two double quotation marks with nothing between them.
+
+```swift
+let hello = "Hello, World!"
+let sausage = "Weißwurst is a tasty sausage."
+var fam = "This is my family: 👨‍👩‍👦‍👦, this is our dog: 🐶"
+var empty = ""
+```
+
+Character literals in Swift look just like string literals, however, only one character is allowed between the quotation marks. Since the two literals look the same, Swift will default to inferring that characters are strings unless the type is specified by an explicit annotation or the context can tell the compiler otherwise.
+
+```swift
+let aString = "A"
+// Swift infers this to be a String
+
+var aChar: Character = "A"
+// Swift now knows to make this a Character
+
+let badChar: Character = "Too many characters"
+// Error: Cannot convert value of type 'String' to specified type 'Character'
+```
+
+Like other constructs in Swift, characters and strings are immutable if defined using `let` and mutable if defined using `var`. So in the above examples, `hello` and `sausage` are immutable strings while `fam` and `empty`, AND `aChar` are mutable.
+
+### Building strings from smaller parts
+
+Strings can be concatenated using the `+` operator, and mutable strings can have other strings appended onto them using the `+=` mutating operator.
+
+```swift
+"honey" + "comb"
+// => "honeycomb"
+
+fam += ", this is our cat: 🐱"
+// => "This is my family: 👨‍👩‍👦‍👦, this is our dog: 🐶, this is our cat: 🐱"
+```
+
+Note that only strings can be concatenated with strings. In order to use concatenation with a character, one must convert it to a String first
+
+```swift
+let question: Character = "?"
+let areYouHappy = "Happy" + question
+// Error: Cannot convert value of type 'Character' to expected argument type 'String'
+
+let areYouHappy = "Happy" + String(question)
+// => "Happy?"
+```
+
+### String Interpolation
+
+The most common way to build up strings in Swift is _string interpolation_. To insert a value into a string using string interpolation, you place an expression or value in parentheses which are preceded by a backslash character.
+
+```swift
+let radius = 5.0
+let interp = "The area of a circle with radius \(radius) is \(Double.pi * radius * radius)"
+// => "The area of a circle with radius 5.0 is 78.53981633974483"
+```
+
+### String and Character equality
+
+Strings and Characters can be compared for equality (or lack thereof) using the `==` operator (or `!=` for not-equal).
+
+```swift
+let hi = "Hello"
+let period: Character = "."
+"\(hi)\(period)" == "Hello."
+// => true
+
+period != ";"
+// => true
```

Comment:
The string interpolation made this example somewhat harder to read to me, as a non-Swift programmer. Maybe also shorten the example a bit:

```suggestion
let hi = "Hello"
hi == "Hello"
// => true

let period: Character = "."
period != ";"
// => true
```